ambry-clustermap/src/main/java/com/github/ambry/clustermap/CloudServiceClusterChangeHandler.java

```diff
@@ -1,157 +0,0 @@
-/*
- * Copyright 2020 LinkedIn Corp. All rights reserved.
```

Comment:
This change is part of https://github.com/linkedin/ambry/pull/2236. Will rebase it once that PR is merged